e2e/credential-revocation.e2e.ts

```diff
@@ -310,15 +335,25 @@ describe('Off-chain credential revocation', () => {
 
     const verifiableCredential = extractCredential(claim);
 
-    mockInitiateCredentialStatusUpdate.mockRejectedValueOnce(
-      new Error('Revoker unauthorized')
-    );
+    mockCredentialInitiate(issuer.signerService.didHex);
 
-    await expect(
-      revoker.verifiableCredentialsService.revokeCredential(
+    const statusListCredential =
+      await revoker.verifiableCredentialsService.revokeCredential(
         verifiableCredential
-      )
-    ).rejects.toThrowError();
+      );
+    expect(statusListCredential).toBeDefined();
+    nock(verifiableCredential.credentialStatus?.statusListCredential as string)
+      .get('')
+      .reply(200, statusListCredential);
+
+    expect(await issuer.claimsService.verifyVc(verifiableCredential)).toEqual(
+      expect.objectContaining({
+        isVerified: false,
+        errors: expect.arrayContaining([
+          expect.stringContaining('StatusList credential is invalid'), // part of error message of InvalidStatusList
```

Comment:
Should we export error messages from ew-credentials to avoid needing to hardcode this string? (would be a separate story)

Reply:
We can get error message from error classes which are already exported. This string appeared because message of https://github.com/energywebfoundation/ew-did-registry/blob/403b8dc790d2abcd36e9ddb56c6e1afca43f12a8/packages/revocation/src/errors.ts#L1 is constructed from more specific messages received from `verify`, which we don't know beforehand. But now I realized it can be achieved as `new InvalidStatusList(['']).message`